### DISCUSSION TITLE
Quarkus Vulnerability in Keycloak:24.0.1

### DISCUSSION THREAD

**Discussion by jsto** [keycloak/keycloak#28096] [Q&A] [answered]
Is Keycloak:24.0.1 which is using quarkus:3.8.1 vulnerable to the following CVEs?

[CVE-2023-6267]
[CVE-2023-5675]

**Comment by abstractj** [comment, marked as answer]
@jsto CVE-2023-6267: the issue was fixed on Quarkus for versions 3.0.0.Final and onwards, up to before 3.2.9.Final, the patched version is 3.2.9.Final. Keycloak actually uses 3.8.3. So we are not vulnerable and not affected.

CVE-2023-5675: CVE-2023-5675 was fixed in Quarkus release 3.2.10.Final​. Keycloak actually uses 3.8.3. So we are not vulnerable and not affected.

**Reply by jsto** [reply]
Thank you @abstractj !
